tests/getting-started-complete-test.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
+    tags: ['getting-started-complete-test'],
+    'Smoke test getting-started-complete' : function (client) {
+      client
+        .url('http://localhost:3000')
+        .pause(1000);
+
+      // Variables for max waiting times
+
+      var apiCallTimeout = 10000;
+      var alertBoxTimeout = 6000;
+      var orderID;
+
+      // Checking that the page loaded
+
+      client.expect.element('body').to.be.present;
+
+      // Checking that the command logs are showing
+
+      client.waitForElementVisible('span.log-string', apiCallTimeout);
+
+      // Checking that the message input is showing
+
+      client.expect.element('input#order').to.be.present;
+
+   
+      // Clicking 'Encrypt' w/o having the function added or a message inputted and checking that an alert-box appears
+
+      client.click('button#encrypt-button');
+      client.waitForElementVisible('div#alert-box', alertBoxTimeout);
+      client.assert.containsText('div#alert-box', 'Please enter an order');
+      client.waitForElementNotVisible('div#alert-box', alertBoxTimeout);
```

Comment:
I'm confused about what this line is doing. You're clicking the encrypt button, then waiting for the alertbox to be visible, asserting about the text. That all makes sense. But why are we waiting for the alertbox to hide since down below you're then setting the input inside the text box.